app/controllers/user.rb

```diff
@@ -0,0 +1,20 @@
+get '/users/new' do
+  @user = User.new
+  erb :"users/new"
+end
+
+post '/users' do
+  @user = User.new(:email => params[:email],
+              :password => params[:password],
+              :password_confirmation => params[:password_confirmation],
+              :first_name => params[:first_name],
+              :last_name => params[:last_name],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.